src/Block/ChildrenPagesBlockService.php

```diff
@@ -172,21 +170,32 @@ public function configureSettings(OptionsResolver $resolver): void
 
     public function load(BlockInterface $block): void
     {
-        if (!$block instanceof PageBlockInterface) {
-            return;
-        }
-
         $pageId = $block->getSetting('pageId', null);
 
-        if (null !== $pageId && !$pageId instanceof PageInterface) {
-            $cmsManager = $this->cmsManagerSelector->retrieve();
+        if (!$pageId instanceof PageInterface) {
```

Comment:
Seems like it's the same here...

Reply:
$this->pageAdmin->getObject($pageId) can return null ; should we throw an exception in that case ?